docs/digger/installation/install-android-sdk.asciidoc

```diff
@@ -0,0 +1,23 @@
+=== Install Android SDK
+
+To build Android applications, Jenkins will create containers (pods) dynamically on OpenShift using the https://wiki.jenkins.io/display/JENKINS/Kubernetes+Plugin[Kubernetes plugin]. This approach has a few advantages:
+
+* It doesn't require a build server to be running all the time
+* Each build is performed completely in isolation
+
+However, Android SDK is required to build Android apps. But it will take too long to build if we have to download the Android SDK every time.
+
+To solve this problem, the installer need to download the Android SDK first and make sure it is available to the building pods. 
+It will perform the following tasks:
+
+* Create a new pod to download the Android SDK. The pod will use an image that contains the required tools, and claims a PV.
+* Invoke a script to download the Android SDK. During the process, the user will be prompted to read the Android SDK license and agree to it. The Android SDK will be persisted to the PV
+* Update the Kubernetes plugin configuration to add a new pod template. It will make sure the PV will be mounted in "Read-Only" mode to the pods and setup the required environment variables to make sure Android home directory is defined.
+
+==== Configure the Android SDK components
+
+The Android SDK contains many different components. 
+Only a subset of them are installed by default. 
+The list of components that will be installed are specified in the `android-sdk/templates/sample_cfg.j2` file of the Digger installer repo.
```

Comment:
For now, we can. But I think it's better to allow users to provide the overrides via Ansible variables. I have created a ticket for that